packages/react/src/LogoSuite/LogoSuite.stories.tsx

```diff
@@ -1,16 +1,32 @@
 import React from 'react'
 
-import {Meta} from '@storybook/react'
-import {LogoSuite} from './LogoSuite'
+import type {Meta, StoryFn} from '@storybook/react'
+import {LogoSuite, type LogoSuiteHeadingProps, type LogoSuiteLogoBarProps, type LogoSuiteProps} from './LogoSuite'
+import {logos} from './LogoSuite.fixtures'
 
-export default {
+type StoryProps = Required<
+  Pick<LogoSuiteProps, 'align' | 'hasDivider'> &
+    Pick<LogoSuiteHeadingProps, 'visuallyHidden'> &
+    Pick<LogoSuiteLogoBarProps, 'marquee' | 'marqueeSpeed' | 'variant'>
```

Comment:
This is cool, better than what's currently here so let's go with it 👍 

Leaving some additional thoughts for posterity too, so we can come back to use of Omit vs Pick.

I could be reading this wrong, but when adding a new prop to LogoSuite we unfortunately won't get it for 'free' here unless we opt in. I think we're doing this because we want to avoid a conflict in names between named children, but that's potentially an edge case. I'd assume in most cases we'd want to benefit from auto opt-in where possible. We could instead manually opt-out those conflicting props as they are really only solving the need to flatten Storybook arguments and nothing to do with the component itself.

Reply:
Yes that's exactly right. I thought this behaviour would be more predictable and intuitive.

> _You want to add a new control to the story? Just add the type to the top of the file too._

As opposed to

> _You want to add a new prop to one of the LogoSuite components? It might cause conflicts in the Storybook types, regardless of whether this prop is actually used in Storybook._

It's just a matter of preference, but I prefer the explicit, opt-in approach to the more magic, opt-out approach. Adding a type to cover new functionality feels much more natural to me than having types be automatically added for me and then potentially having to manage conflicts. If we adopt this approach everywhere then conflicts might not be that uncommon as we reuse prop names fairly regularly across components.